oak_functions/loader/src/lookup.rs

```diff
@@ -163,3 +178,36 @@ pub fn parse_lookup_entries<B: prost::bytes::Buf>(
     }
     Ok(entries)
 }
+
+async fn request_bytes<C>(client: &Client<C, Body>, request: Request<Body>) -> anyhow::Result<Bytes>
```

Comment:
How about `response_bytes` or `send_request`?

Reply:
Renamed to `send_request`.